### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by ppfeister**
Adds requirement.txt not required note, as discussed in #2156.
Adjusts install options in Readme, including the addition of Kali/Parrot/BlackArch.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -30,8 +30,11 @@
 | | Command | Notes |
 | - | - | - |
 | PyPI | `pipx install sherlock-project` | `pip` may be used in place of `pipx` |
-| Homebrew | `brew install sherlock` | Community supported |
 | Docker | `docker pull sherlock/sherlock` | |
+| Debian family | `apt install sherlock` | Kali, Parrot, BlackArch, Testing, Sid |
```
Comment (sdushantha) [inline docs/README.md:34]: BlackArch is apart of Arch Linux and not Debian

> BlackArch Linux is an **Arch Linux-based** penetration testing distribution for penetration testers and security researchers. 

Source: https://blackarch.org/

**Review by sdushantha — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (ppfeister) [reply docs/README.md:34]: Agh. Working on autopilot. Will adjust.

Comment (ppfeister) [reply docs/README.md]: Removed BlackArch

File: docs/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@
 | - | - | - |
 | PyPI | `pipx install sherlock-project` | `pip` may be used in place of `pipx` |
 | Docker | `docker pull sherlock/sherlock` | |
-| Debian family | `apt install sherlock` | Kali, Parrot, BlackArch, Testing, Sid |
+| Debian family | `apt install sherlock` | Kali, Parrot, Deb Testing, Deb Sid |
```
Comment (sdushantha) [inline docs/README.md:34]:
```suggestion
| Debian family | `apt install sherlock` | Kali, Parrot, Debian and Sid |
| BlackArch | `pacman -S sherlock` |  |
```

**Review by sdushantha — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (ppfeister) [reply docs/README.md:34]: Works for me! Couldn't remember if BlackArch used pacman as well or if they used something special

**Review by sdushantha — APPROVED** [review]

**Pull request merged**